README.md

```diff
@@ -1,55 +1,45 @@
-  [RosBridge Protocol](https://github.com/RobotWebTools/rosbridge_suite/blob/ros1/ROSBRIDGE_PROTOCOL.md)
-- Overview: [https://foxglove.dev/images/blog/using-rosbridge-with-ros2/hero.webp](https://foxglove.dev/images/blog/using-rosbridge-with-ros2/hero.webp)
-- [Roslibjs source code]: https://github.com/RobotWebTools/roslibjs
+### **For Software Members not working on mission control**
 
-## Recommended IDE Setup
+- **Testing Locally (on personal Docker):**
+  - Visit [Mission Control](https://www.trickfirerobotics.org/mission-control/) and follow the URC instructions to run the Docker container. Click on the Navbar websocket icon to switch to local development
+- **Running on the Rover:**
+  - Visit [Mission Control](https://www.trickfirerobotics.org/mission-control/) and follow the **Launching on Rover** instructions below.
 
-- [VSCode](https://code.visualstudio.com/)
+### **For Non-Software Members**
 
-- [Vue DevTools Chrome Extension](https://devtools.vuejs.org/getting-started/installation): Can Inspect Components State and integrated with Pinia, allowing to see store's state
+Visit [Mission Control](https://www.trickfirerobotics.org/mission-control/) and follow the **Launching on Rover** instructions below.
 
-## Project Setup
+## **Launching on Rover**
 
-1. Install Node.js version 22.11.0 (LTS) (Long-term support Updated 11/6/2024)
-   - **Windows and macOS**:
-   1. If different node version is already installed, delete it from your system or if using version manager, switch to it.
-      - **Windows**: https://stackoverflow.com/questions/20711240/how-to-completely-remove-node-js-from-windows (Tested 11/6/2024)
-   2. [Download version 22.11.0 (LTS)](https://nodejs.org/en/download/prebuilt-installer) and run the installer
-   - **Linux**: [Follow Instructions to install 22.11.0 (LTS)](https://nodejs.org/en/download/package-manager) (Not tested)
-2. Run `git clone https://github.com/TrickfireRobotics/mission-control.git` in terminal **(not in the URC Docker Container)**
-3. Run `cd mission-control` to open the repository
-4. Run `npm install` to install dependencies
+Setup router and ensure connect to trickfirerouter on your computer. The router password is on the router itself Turn the rover computer on by plugging in the c port charger. You will know the computer is on if the LED lights up.
+Run the commands below in Powershell or in your OS terminal. When prompted for a password, enter `trickfire`
 
-### Launching
-
-1. Run `npm run dev`
-2. While running open another VSCode window to the URC repo in the docker environment
-   - Run `./build.sh` and `./launch.sh` commands in the container terminal
-3. If the Websocket circle becomes **green**, you are connected!
+```
+Open Powershell or Terminal
+> ssh trickfire@192.168.0.145
+In Bash Now
+> cd ~/urc-2023
+> sudo ./container_launch.sh
+In Docker Now
+> cd /home/trickfire/urc-2023
+> ./build.sh
+> ./launch.sh
```

Comment:
This feels like something that should be included in the urc-2023 repo, or as internal documentation. It's specific to the configuration on the rover, and doesn't really have that much to do with mission-control.

Reply:
Fixed all of the typos and grammar stuff. Moved this particular one to TrickfireRobotics/urc-2023#81